build.gradle

```diff
@@ -25,13 +25,13 @@ repositories {
 
 
 dependencies {
-    compileOnly 'org.projectlombok:lombok'
```

Comment:
Po co te wszystkie zmiany w dependency? Zmiana wersji jest ok, ale raczej nie mieszajmy z kolejnością dependency jak już jakaś była bo tylko doda to konflikty komuś. Poza tym nie potrzebnie jest dodane:
//compile'org.apache.tomcat.embed:tomcat-embed-jasper'

Reply:
build.gardle w ogóle nie  miał iść do commita, mój błąd